### ISSUE TITLE
Aligning nodes horizontally 

### ISSUE THREAD

**Issue by RehabHussien** [kieler/elkjs#169]
Hi, I've this configuration:
```
	const graph = {
			id: "root",
			layoutOptions: {
				"elk.algorithm": "mrtree",
				"elk.direction": "DOWN",
				"elk.spacing.nodeNode": "130",
				"mrtree.weighting": "FAN",
				"org.eclipse.elk.layered.nodePlacement.bk.fixedAlignment": "RIGHTUP",
				"elk.spacing.componentComponent": "10000",
				"elk.aspectRatio": 100,
			},
		};
```
I've a variable nodes heights , my question is, how to align the nodes horizontally to top, cannot figure out which configuration responsible for this.
<img width="867" alt="Screen Shot 2022-05-20 at 7 35 56 AM" src="https://user-images.githubusercontent.com/82047913/169457874-f78d823c-8e9e-4a5f-8097-d007648f9c71.png">
The image shows that they are centered but I need them to be aligned to top, any help in that

**Comment by soerendomroes** [comment]
Have you tried to set an [alignment](https://www.eclipse.org/elk/reference/options/org-eclipse-elk-alignment.html)?

**Comment by RehabHussien** [comment]
yeah I tried setting the alignment, well can I achieve the same result for the x position with "layered" , I need the parent to be centered between the children and wasn't able to achieve that except with `mrtree`

**Comment by soerendomroes** [comment]
Have you tried using a different nodePlacement.strategy? 
https://www.eclipse.org/elk/reference/options/org-eclipse-elk-layered-nodePlacement-strategy.html

**Comment by RehabHussien** [comment]
@soerendomroes yes I tried all different types but failed to get the required output of centered parent and top aligned children

**Comment by soerendomroes** [comment]
Okay, this leaves two different potential feature request you could file in [ELK](https://github.com/eclipse/elk):
- Add alignment to MrTree
- Add a centering node placement strategy to the layered algorithm
